### DISCUSSION TITLE
ImportError when using XTTS(Coqui)

### DISCUSSION THREAD

**Discussion by arunman1kandan** [pydantic/pydantic#10458] [Question]
Hey guys , I am using python 3.9 and pydantic version : 2.9.2 and whenever I try to run XTTS I get the following error:

![image](https://github.com/user-attachments/assets/7b7bd5ea-af87-4f4a-9ca2-572f52d38dd1)

Any idea what is the issue I have tried uninstalling and installing still the issue persists.

Thanks in Advance
~Arun Manikandan

**Comment by Viicos** [comment]
You should probably upgrade your `thinc` and/or `confection` dependencies.
